### PR TITLE
Updates setup.py script and support for python3

### PR DESCRIPTION
**Pull request by EiffL**
This PR updates the setup.py script so that the package can now be pip installable. To try it out:
```
$ pip install git+https://github.com/EiffL/Stile
```
Once this PR is approved, @msimet you can create an account on PyPi and the package will be automatically uploaded for you by Travis CI everytime you cut a new release.

I've also tried to convert (automatically) the package to Python3 but I'm not runinng all the tests, so not sure how much of the code is being covered

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - "3.6"
+
+install:
+  - pip install .
+
+script:
+  - nosetests tests/test_binning.py tests/test_stile_utils.py
+
+after_success:
+- coveralls
```
[the remark this answers is not in the document]
Comment (EiffL) [reply .travis.yml]: Done and done

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Stile
+Stile [![Build Status](https://travis-ci.org/EiffL/Stile.svg?branch=master)](https://travis-ci.org/EiffL/Stile)
```
[the remark this answers is not in the document]
Comment (EiffL) [reply README.md]: and done :-)

**Comment by msimet** [comment]
Awesome, thank you so much!

**Comment by EiffL** [comment]
and I broke it ***facepalm*** added install of coverage and changed the path to the tests. Note that I'm only running 2 tests here, to be updated I guess when your other PR with updated tests is merged

**Comment by msimet** [comment]
Yes, I did notice that! I'll add more tests to the PR for #103, thanks.

**Pull request merged**